components/standings/commons/standings_storage.lua

```diff
@@ -240,8 +244,14 @@ function StandingsStorage.fromTemplateEntry(frame)
 
 	data.opponent = Opponent.resolve(Opponent.readOpponentArgs(opponentArgs) or Opponent.tbd(), date)
 
-	-- Template don't have SlotIndex, use placement as workaround
-	data.slotindex = data.placement
+	if (data.placement or ''):lower() == DISQUALIFIED then
+		data.definitestatus = DISQUALIFIED
+		data.currentstatus = DISQUALIFIED
+	end
+
+	-- If template doesn't have SlotIndex, use placement as workaround
+	data.slotindex = tonumber(data.slotindex) or tonumber(data.placement)
+	data.placement = tonumber(data.placement) or data.slotindex
```

Comment:
possibly add comment along the lines of `-- If placement is not numeric use slotindex to avoid storing 0`

alternatively just do
```lua
 	data.placement = tonumber(data.placement)
 	-- If template doesn't have SlotIndex, use placement as workaround
 	data.slotindex = tonumber(data.slotindex) or data.placement
```

mind the slotindex would have to be passed along by all the group slot templates in both versions
(bot run?!)

Reply:
yes, but the .entry one sets it via slotindex

last sentence: [this](https://liquipedia.net/counterstrike/index.php?title=Template:GroupTableSlot&diff=2549876&oldid=2338665) for all wikis

Reply:
i didn't know they exist

hmmm, maybe doing var stuff might be a good idea after all

mind that for swiss the placeMapping would be needed too

Reply:
should work, but you also would need to reset it to 0 in the group start templates
(for the case of having >1 group on the page)

Reply:
ah yeah, would work